test/sanity/ignore.txt

```diff
@@ -6,10 +6,13 @@ lib/ansible/executor/powershell/async_wrapper.ps1 pslint:PSCustomUseLiteralPath
 lib/ansible/executor/powershell/exec_wrapper.ps1 pslint:PSCustomUseLiteralPath
 lib/ansible/galaxy/collection/__init__.py mypy-3.10:attr-defined  # inline ignore has no effect
 lib/ansible/galaxy/collection/__init__.py mypy-3.11:attr-defined  # inline ignore has no effect
+lib/ansible/galaxy/collection/__init__.py mypy-3.12:attr-defined  # inline ignore has no effect
```

Comment:
This file should also include a skip entry for Python 3.12 like the one here:

https://github.com/ansible/ansible/blob/e964078a83af240b103e34eafca9162deff0e16f/test/sanity/ignore.txt#L51

That will avoid sanity test errors on systems without libselinux present.

Reply:
There isn't one for 3.11 or 3.10, so I didn't add one for 3.12. Should I add those as well? https://github.com/ansible/ansible/blob/e964078a83af240b103e34eafca9162deff0e16f/test/sanity/ignore.txt#L51-L55.

Reply:
I already added the missing 3.10 and 3.11 entries in https://github.com/ansible/ansible/pull/81305